odl/tomo/backends/astra_cuda.py

```diff
@@ -281,7 +281,8 @@ def call_backward(self, proj_data, out=None):
         out[:] = self.out_array
 
         # Fix inconsistent scaling in ASTRA
```

Comment:
Can we remove these comments? It's largely been reduced to different conventions, no? No more need to be cocky.